testsuite/ui/widgets/search.py

```diff
@@ -0,0 +1,20 @@
+"""Common elements for search form"""
+
+# Following ignore is needed otherwise: Method 'fill' is abstract in class 'Widget' but is not overridden ...
+# pylint: disable=abstract-method
+
+from widgetastic.widget import GenericLocatorWidget, TextInput
+
+
+class SearchInput(TextInput):
+    """Widget representing 3scale search input used on several places"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, locator="//input[@aria-label='Search input']", **kwargs)
+
+
+class SearchButton(GenericLocatorWidget):
```

Comment:
Please move the search button to buttons module and ideally Input to general widget module

Reply:
Or alternatively, search related items could be bundled together. None of these is supposed to live without the other one.

Reply:
I created new SearchInput widget following model of widgetastic that couples required features of PF4 search-input